impls/src/backends/lmdb.rs

```diff
@@ -515,7 +515,7 @@ where
 		// Save the previous output data to the db.
 		if let Ok(previous_output) = self.get(&out.key_id, &out.mmr_index) {
 			if previous_output != out {
-				self.save_output_history(previous_output);
+				let _ = self.save_output_history(previous_output);
```

Comment:
Why this?

Reply:
The function returns a Result and the cargo asks to use that result, but as the code itself didn't need it, I just made the cargo accept using a `let _ `